Sources/SwiftKueryORM/Model.swift

```diff
@@ -21,1126 +21,1071 @@ import Dispatch
 
-  /// Call to find all the models in the database matching the QueryParams that accepts a completion
-  /// handler. The callback is passed an array of tuples (id, model) or an error
-  static func findAll<Q: QueryParams, I: Identifier>(using db: Database?, matching queryParams: Q?, _ onCompletion: @escaping ([(I, Self)]?, RequestError?) -> Void)
+    /// Call to find all the models in the database matching the QueryParams that accepts a completion
+    /// handler. The callback is passed an array of tuples (id, model) or an error
+    static func findAll<Q: QueryParams, I: Identifier>(using db: Database?, matching queryParams: Q?, _ onCompletion: @escaping ([(I, Self)]?, RequestError?) -> Void)
 
-  /// Call to find all the models in the database matching the QueryParams that accepts a completion
-  /// handler. The callback is passed a dictionary [id: model] or an error
-  static func findAll<Q: QueryParams, I: Identifier>(using db: Database?, matching queryParams: Q?, _ onCompletion: @escaping ([I: Self]?, RequestError?) -> Void)
+    /// Call to find all the models in the database matching the QueryParams that accepts a completion
+    /// handler. The callback is passed a dictionary [id: model] or an error
+    static func findAll<Q: QueryParams, I: Identifier>(using db: Database?, matching queryParams: Q?, _ onCompletion: @escaping ([I: Self]?, RequestError?) -> Void)
 
 }
 
 public extension Model {
-  /// Defaults to the name of the model + "s"
-  static var tableName: String {
-    let structName = String(describing: self)
-    if structName.last == "s" {
-      return structName
-    }
-    return structName + "s"
-  }
-
-  /// Defaults to "id"
-  static var idColumnName: String { return "id" }
-  /// Defaults to Int64
-  static var idColumnType: SQLDataType.Type { return Int64.self }
-
-  @discardableResult
-  static func createTableSync(using db: Database? = nil) throws -> Bool {
-    var result: Bool?
-    var error: RequestError?
-    let semaphore = DispatchSemaphore(value: 1)
-    createTable(using: db) { res, err in
-      result = res
-      error = err
-      semaphore.signal()
-    }
-    semaphore.wait()
-
-    if let errorUnwrapped = error {
-      throw errorUnwrapped
-    }
-    guard let resultUnwrapped = result else {
-      throw RequestError(.ormInternalError, reason: "Database table creation function did not return expected result (both result and error were nil)")
-    }
-
-    return resultUnwrapped
-  }
-
-  static func createTable(using db: Database? = nil, _ onCompletion: @escaping (Bool?, RequestError?) -> Void) {
-    var connection: Connection
-    var table: Table
-    do {
-      connection = try Self.getConnection(using: db)
-      table = try Self.getTable()
-    } catch let error {
-      onCompletion(nil, Self.convertError(error))
-      return
+    /// Defaults to the name of the model + "s"
```

Comment:
Because of the indentation change, it's really hard to tell what's actually changed here... is it possible to do it as a separate PR?

My concern is that this is a `public extension` which - I _think_ - means that all its members are by default public.  Any changes here would therefore be to the public API.